tools/stress/src/org/apache/cassandra/stress/Stress.java

```diff
@@ -27,6 +30,9 @@
 import org.apache.cassandra.utils.FBUtilities;
 import org.apache.cassandra.utils.WindowsTimer;
 
+import sun.misc.Signal;
+import sun.misc.SignalHandler;
```

Comment:
Are the `sun.misc.*` inside the `internal` parts of JDK and shouldn't be accessed?

Reply:
Yes, but outside of shutdown hooks I don't know of a way to do this. If it ever gets removed I assume there will be something else for signal handling and in that case this change should be easy to port or ditch altogether since it does not go too deep.

Reply:
You are right, there is no other way to get to the Unix signals, and handle specific signal through Java Public APIs